data-exporter/src/main/java/org/reactome/release/Utilities.java

```diff
@@ -0,0 +1,166 @@
+package org.reactome.release;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+/**
+ * General Utilities class.  This class contains methods needed for general manipulation of data or common operations
+ * used by the data export process.
+ * @author jweiser
+ */
+public class Utilities {
```

Comment:
Possibly rename to `DataExportUtilities`, if the functionality within is only useful for this data-export project? Or would it be useful in more general projects? Are any of these functions useful enough that they could go into release-common-lib?

Reply:
No problem -- I'll rename it.  There are some general methods that could be re-used, but I'd like to keep them here until they're needed in more than one place in the data-release-pipeline.  I've created an issue just to keep in mind the methods can be moved to release-common-lib if they are needed elsewhere.

https://github.com/reactome/data-release-pipeline/issues/49